makoctl

```diff
@@ -35,20 +37,21 @@ fi
 
 case "$1" in
 "dismiss")
-	[ $# -lt 2 ] && action="" || action="$2"
-	case "$action" in
+	[ $# -lt 2 ] && arg="0" || arg=$2
+	case "$arg" in
 	"-a"|"--all")
 		call DismissAllNotifications
 		;;
 	"-g"|"--group")
 		call DismissGroupNotifications
 		;;
-	"")
-		call DismissLastNotification
-		;;
 	*)
-		echo "makoctl: unrecognized option '$2'"
-		exit 1
+		re='^[0-9]+$'
```

Comment:
I don't think it's worth it to try to validate the ID, `busctl` will do it anyway. For instance:

```
Failed to parse 'asdf' as unsigned 32bit integer: Invalid argument
```